### DISCUSSION TITLE
Installation on Hosting Space

### DISCUSSION THREAD

**Discussion by ashaqashraf** [TareqMonwer/Django-School-Management#135] [Q&A]
Can I use this app on my online hosting space of GoDady

Any kind of help is highly appreciable

**Comment by TareqMonwer** [comment]
> Can I use this app on my online hosting space of GoDady
> 
> Any kind of help is highly appreciable

Hi!
Surely you can deploy and use this project. A courtesy/accredition will be highly appreciated.
